pkg/app/app_reconcile.go

```diff
@@ -129,6 +140,9 @@ func (a *App) reconcileFetchTemplateDeploy() exec.CmdRunResult {
 			UpdatedAt: metav1.NewTime(time.Now().UTC()),
 		}
 
+		a.timeMetrics.RegisterFetchTime(a.app.Kind, a.app.Name, a.app.Namespace, a.isFirstReconcile,
```

Comment:
since we have already registered fetch time inside a.fetch(...) function, we should not do this. else it will overwrite previous value.